modules/widgets/authors.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/*
+ * Widget to display blog authors with avatars and recent posts.
+ *
+ * Configurable parameters include:
+ * 1. Whether to display authors who haven't written any posts
+ * 2. The number of posts to be displayed per author (defaults to 0)
+ * 3. Avatar size
+ */
+class Widget_Authors extends WP_Widget {
+	public function __construct() {
+		parent::__construct(
+			'authors',
+			/** This filter is documented in modules/widgets/facebook-likebox.php */
+			apply_filters( 'jetpack_widget_name', __( 'Authors', 'jetpack' ) ),
+			array( 'classname' => 'widget_authors', 'description' => __( 'Display blogs authors with avatars and recent posts.', 'jetpack' ) ),
+			array( 'width' => 300 )
```

Comment:
Maybe we should add selective refresh, like we've done for all other new widgets.